login/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓로그인</title>
+    <link rel="stylesheet" href="/login/login.css" />
+  </head>
+  <body>
+    <div class="login_container">
+      <a href="/" div class="login_logo">
+        <img src="/assets/icons/판다 얼굴.png" />
+        <span class="logo_text">판다마켓</span>
+      </a>
+      <div class="login_frame">
```

Comment:
여기도 엘리먼트의 역할이 좀더 명확히 드러날수있게 바꿔볼까요?

```suggestion
      <form class="login_form_container">
```

input form field들을 담고있는 컨테이너니까, div보단 form 태그를 사용해주시는게 나중에 서버에 폼데이터 제출할때도 편리할거예요!

이렇게 수정된다면 아래 `login_textbox`는 `login_form`정도가 적당하겠죠? :)